docs/git/index.md

```diff
@@ -0,0 +1,295 @@
+
+There's another file in the repository you've cloned, `other-file.txt`. Open it, answer the question in the file, commit, and push.
+
+Notice a couple of things are different this time!
+
+It didn't ask you to fork the repository - that's because you already have a fork you're allowed to push to.
+
+You may also notice that if you click the "Create Pull Request" menu item, it brings you to a different page. It doesn't have a "Create Pull Request" button, it has a "View Pull Request" button. This is because if you already have a pull request open, when you push more changes, it will update your existing pull request.
+
+This is useful for responding to feedback you get on your homework. There are ways to open more than one pull request at a time (using something called branches), which we'll learn about in the future, but for now, one should do!
+
+## Getting feedback
+
+When you've made your pull request, our volunteers will be notified. They will look at your changes. When they are done, you will get an email. Here are some of the things they may do:
+
+- Make comments with suggestions, either about a particular bit of code, or about the whole pull request.
+- Add labels to the pull request (e.g. marking it as complete, or unfinished).
+
+If they give you suggestions, you should try to implement the suggestions, and push a new commit. If you're confused, struggling, or find them unclear, you can respond to the comment with a comment of your own, or ask on Slack.
+
+## History
+
+One useful features of Git is how it stores your commits. In GitHub Desktop, if you open the History tab, you can see a list of each commit that's been made in the repository, with the oldest at the bottom and the newest at the top.
+
+If you click on one of the commits, you'll see the changes that happened in it. This can be really useful to understand how the repository evolved to how it looks today. It can also help us to find out when bugs were introduced!
+
+You can always see an old version of a file by looking in the Git history, and if you want to get it back, you can just copy and paste it from the history view into your text editor.
+
+### When to commit, push, and make a pull request
+
+You should commit often! Every time you think you've done something you may want to look at again, you should make a commit.
+
+Let's say you've made a website, and it generally looks about right, but you were thinking of adding some colour, or an animation. Before you do that, make a commit, because then if you break some of the CSS by trying to add an animation, you can always undo it.
+
+Or if you want to try out a few different colours, make a commit for each colour, and then you can easily see what colours you tried out, and compare them.
+
+If you don't commit, maybe you'll try the perfect colour, then try a different colour, and forget what the previous colour was! Disaster! But if you commit each thing you try, you can always go back and see what it was!
```

Comment:
Maybe this would be good as a teacher-led demo?

Reply:
Drafted up - need to make a small website to show as an example